e2e/support/helpers/e2e-api-key-helpers.ts

```diff
@@ -0,0 +1,6 @@
+export const createApiKey = (name: string, group_id: number) => {
+  return cy.request("POST", "/api/api-key", {
+    name,
+    group_id,
+  });
+};
```

Comment:
extracted this little helper to a common helpers file since we're using it in multiple files now